pkg/approver/attribute/attribute.go

```diff
@@ -30,15 +31,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
-	cmpapi "github.com/cert-manager/policy-approver/apis/policy/v1alpha1"
-	"github.com/cert-manager/policy-approver/internal/pkg/attribute/checks"
-	"github.com/cert-manager/policy-approver/registry"
+	cmpapi "github.com/cert-manager/policy-approver/pkg/apis/policy/v1alpha1"
+	"github.com/cert-manager/policy-approver/pkg/approver"
+	"github.com/cert-manager/policy-approver/pkg/approver/attribute/internal"
+	"github.com/cert-manager/policy-approver/pkg/registry"
 )
 
-// Load the attribute evaluator checks.
-func init() {
-	registry.Load(evaluateBaseChecks)
-}
+var _ approver.Interface = attribute{}
```

Comment:
I'm not sure what this is doing. Can you explain it - perhaps even in a comment?

Reply:
It is a compile time check to make sure `attribute` implements `approver.Interface`.

https://medium.com/@matryer/golang-tip-compile-time-checks-to-ensure-your-type-satisfies-an-interface-c167afed3aae

Reply:
Ok thanks.

Reply:
(don't seem to have a button to resolve.)